app/jobs/office_email_application_job.rb

```diff
@@ -1,14 +1,24 @@
 class OfficeEmailApplicationJob < ApplicationJob
   def perform(export:)
-    export.execute do |snap_application, logger|
-      ApplicationMailer.office_application_notification(
-        file_name: snap_application.pdf.path,
-        recipient_email: snap_application.receiving_office_email,
-        office_location: snap_application.office_location,
-      ).deliver
+    export.execute do |benefit_application, logger|
+      if benefit_application.class == SnapApplication
```

Comment:
Feels a bit odd that perform is one big conditional. My hunch is that this should be two different job classes.

Reply:
ok we can do that! I was hoping that the job could run the same email with different view templates but this somehow felt clearer to me. what do you think?

Reply:
Would a shared partial solve that?

Reply:
yeah we already have different partials used for the mailer depending on whether the SNAP app has on `office_location` so we could have another branch of view partial logic for Medicaid apps....let me put something together once Christa and I are done with our current work and circle back with a solution that uses a different partial in the mailer rather than logic here

Reply:
moved to its own class /job rather than a conditional at cc1a53e9